appengine/gophers/gophers-5/index.html

```diff
@@ -0,0 +1,113 @@
+<!doctype html>
+<html>
+<head>
+  <title>The Gopher Network</title>
+
+  <link type="text/css" rel="stylesheet" href="/static/style.css" />
+
+  <!-- See https://github.com/firebase/firebaseui-web. -->
+  <!-- [START init_firebase] -->
+  <!-- *******************************************************************************************
+       * TODO(DEVELOPER): Paste the initialization snippet from:
+       * Firebase Console > Overview > Add Firebase to your web app.
+       ***************************************************************************************** -->
+  <!-- [END init_firebase] -->
+
+  <!-- [START include_firebaseui] -->
+  <script src="https://cdn.firebase.com/libs/firebaseui/2.6.2/firebaseui.js"></script>
+  <link type="text/css" rel="stylesheet" href="https://cdn.firebase.com/libs/firebaseui/2.6.2/firebaseui.css" />
+  <!-- [START include_firebaseui] -->
+  <!-- [START handle_login] -->
+  <script>
+    initApp = function() {
+      document.getElementById('sign-out').onclick = function() {
+        firebase.auth().signOut();
+      };
+      // FirebaseUI config.
+      var uiConfig = {
+        signInSuccessUrl: '/',
+        signInOptions: [
+          // Leave the lines as is for the providers you want to offer your users.
+          firebase.auth.GoogleAuthProvider.PROVIDER_ID,
+          firebase.auth.FacebookAuthProvider.PROVIDER_ID,
+          firebase.auth.TwitterAuthProvider.PROVIDER_ID,
+          firebase.auth.GithubAuthProvider.PROVIDER_ID,
+          firebase.auth.EmailAuthProvider.PROVIDER_ID,
+          firebase.auth.PhoneAuthProvider.PROVIDER_ID
+        ],
+        // Terms of service url.
+        tosUrl: '<your-tos-url>'
+      };
+
+      firebase.auth().onAuthStateChanged(function(user) {
+        if (user) {
+          // User is signed in.
+          document.getElementById('sign-out').hidden = false;
+          document.getElementById('post-form').hidden = false;
+          var account = document.getElementById('account-details').textContent =
+            "Signed in as " + user.displayName + " (" + user.email + ")";
```

Comment:
single quotes for consistency. 4 space indent for continuations. (go/jsstyleguide)

Reply:
Done.